beacon-chain/main.go

```diff
@@ -111,6 +112,14 @@ func main() {
 	app.Flags = appFlags
 
 	app.Before = func(ctx *cli.Context) error {
+		// verify if ToS accepted
+		if accepted, err := tos.VerifyTosAcceptedOrPrompt(ctx); accepted != true {
+			if err != nil {
+				return err
+			}
+			return fmt.Errorf("you have to accept TERMS AND CONDITIONS in order to continue")
+		}
```

Comment:
I know it's probably not possible in your current implementation, but it would be safer to handle these values individually. If VerifyTosAcceptedOrPrompt ever returned true and an error, then your check would happily accept it. This could happen by misuse or accidental changes in the future. 

Better to be safe than sorry, I think!

```suggestion
		accepted, err := tos.VerifyTosAcceptedOrPrompt(ctx); 
		if err != nil {
			return err
		}
		if !accepted {
			return errors.New("you have to accept Terms and Conditions in order to continue")
		}
```